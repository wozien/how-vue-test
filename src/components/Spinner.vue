<template>
  <transition>
    <svg class="spinner" width="44px" height="44px" viewBox="0 0 44 44">
      <circle class="path" fill="none" stroke-width="4" stroke-linecap="round" cx="22" cy="22" r="20"></circle>
    </svg>
  </transition>
</template>