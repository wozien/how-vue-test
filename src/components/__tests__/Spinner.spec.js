import { shallowMount } from '@vue/test-utils'
import Spinner from '../Spinner.vue'

describe('Spinner', () => {
  test('render correctly', () => {
    expect(shallowMount(Spinner).element).toMatchSnapshot()
  })
})