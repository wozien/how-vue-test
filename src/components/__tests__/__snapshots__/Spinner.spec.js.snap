// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner render correctly 1`] = `
<transition-stub>
  <svg
    class="spinner"
    height="44px"
    viewBox="0 0 44 44"
    width="44px"
  >
    <circle
      class="path"
      cx="22"
      cy="22"
      fill="none"
      r="20"
      stroke-linecap="round"
      stroke-width="4"
    />
  </svg>
</transition-stub>
`;
